### DISCUSSION TITLE
How to show list from db in custom page

### DISCUSSION THREAD

**Discussion by iranandroid** [filamentphp/filament#5135] [Help] [answered]
Hi everyone, i'm new in Laravel and Filament. i created custom page and i wanna show list of data from db, but i don't know how to.
I have list of banners and countries, now i wanna set limitation to show banner in countries.
I need show list of countries and admin can enable country to show banner.

Can you help me?

**Comment by danharrin** [comment, marked as answer]
You'll need some knowledge of Livewire, as custom pages are Livewire components
